TSOClient/tso.client/UI/Profile/UIJobInfo.cs

```diff
@@ -0,0 +1,377 @@
+            PromotionPercentage = (int)(promotionPercentage * 100);
+            MaxLevel = (jobGrade == 10);
+        }
+    }
+
+    public class UIJobInfo : UIAlert
+    {
+        private new UIButton OKButton;
+
+        private UILabel Title;
+        
+        private UILabel Type;
+        private UILabel Level;
+        private UILabel Hours;
+        private UILabel CarPoolHours;
+        private UILabel NextPosition;
+        private UIProgressBar ProgressBar;
+
+        private UILabel PositionTitle;
+        private UILabel HoursTitle;
+        private UILabel PerformanceTitle;
+        private UILabel NextPositionTitle;
+
+        private UILabel PromotionRequirements;
+
+        private JobInformation jobInformation;
+
+        public UIJobInfo(JobInformation jobInformation) : base(new UIAlertOptions()
+        {
+            Title = "Job Details",
+            Width = 400,
+            Height = jobInformation.MaxLevel ? 200 : 348,
+            Message = "",
+            Buttons = new UIAlertButton[]
+            {
+                new UIAlertButton(UIAlertButtonType.OK, (btn) => { }),
+            },
+            AllowBB = true
+        })
+        {
+
+            this.jobInformation = jobInformation;
+
+            int standardVerticalSpace = 10;
+            int verticalSpace = 40;
+            int sectionHeight = 54;
+
+            OKButton = ButtonMap[UIAlertButtonType.OK];
+            OKButton.OnButtonClick += (btn) => Destory();
+
+            Type = new UILabel
+            {
+                Position = new Vector2(30, verticalSpace),
+                Size = new Vector2(200, 16),
+                CaptionStyle = TextStyle.DefaultTitle
+            };
+            Type.CaptionStyle = Type.CaptionStyle.Clone();
+            Type.CaptionStyle.Color = Color.White;
+            Type.CaptionStyle.Size = 16;
+            Type.CaptionStyle.Shadow = true;
+            Type.Alignment = TextAlignment.Left | TextAlignment.Middle;
+            this.Add(Type);
+            verticalSpace += 16 + standardVerticalSpace;
+
+            UIImage HoursSectionImage = new UIImage(GetTexture((ulong)0x7A400000001)).With9Slice(13, 13, 13, 13);
+            HoursSectionImage.SetSize(360, sectionHeight);
+            HoursSectionImage.Position = new Vector2(20, verticalSpace);
+            Add(HoursSectionImage);
+            this.Add(HoursSectionImage);
+            verticalSpace += standardVerticalSpace;
+
+            int hoursTitleSectionHeight = (sectionHeight / 2) - 12 - 6;
+
+            HoursTitle = new UILabel
+            {
+                Position = new Vector2(40, verticalSpace + hoursTitleSectionHeight),
+                Size = new Vector2(200, 12),
+                CaptionStyle = TextStyle.DefaultTitle
+            };
+            HoursTitle.CaptionStyle = HoursTitle.CaptionStyle.Clone();
+            HoursTitle.CaptionStyle.Color = new Color(238, 247, 169);
```

Comment:
No need setting default values - the yellow colour and shadow false are default, so they do not need to be reset when cloning the default title.

Reply:
Removed setting unnecessary properties on yellow text non-shadow items.